### ISSUE TITLE
Fishy quality of decoded m4a/aac files

### ISSUE THREAD

**Issue by ZeroOnions** [asivery/ElectronWMD#12]
Probably not strictly ElectronWMD issue, but it's the only version that I can get connected to my N920.

If I take any of my m4a music files (doesn't matter lossless or lossy), convert it to wav and then transfer to MD, it'll sound noticeably better than when i put it directly to EWMD and let it do the decoding itself. 

I'm using it on macOS Ventura 13.1 at the moment

**Comment by asivery** [comment]
That's very interesting, how are you converting the files to WAV? ElectronWMD uses ffmpeg underneath, so that might be the reason for the quality problems. Would you mind sharing the m4a and wav files for me to compare?

**Comment by ZeroOnions** [comment]
I'm sorry @asivery - I can't reliably recreate this at the moment. I remember putting first or second Portishead album on a disc, first from my m4a files directly, and later trying one or two tracks decoded first (using ocenaudio --> save as --> wav 16bit no dithering). There were some hard hats/cymbals in those tracks and I remember them sounding very bad at first, and then better if I decoded them outside the app. However - I tried now and I can't hear that difference. Either it was completely in my head, or something has changed since then. Anyway - i'm closing it for now, if I catch something like that again, I'll grab test files immediately and create new issue. Cheers!
